src/metabase/task.clj

```diff
@@ -265,7 +265,7 @@
    :priority           (.getPriority trigger)
    :start-time         (.getStartTime trigger)
    :may-fire-again?    (.mayFireAgain trigger)
-   :data               (.getJobDataMap trigger)})
+   :data               (into {} (.getJobDataMap trigger))})
```

Comment:
Why did you need to do this?

Reply:
we need this in tests because we can't compare JobDataMap with clojure map